guides/common/modules/con_using-boot-disks-to-provision-hosts.adoc

```diff
@@ -1,4 +1,10 @@
 [id="using-boot-disks-to-provision-hosts"]
 = Using boot disks to provision hosts
 
-You can use boot disks to boot hosts without PXE firmware.
+Some hardware does not provide a PXE boot interface.
+In {Project}, you can provision a host without PXE boot.
+This is also known as PXE-less provisioning and involves generating a boot ISO that hosts can use.
+Using this ISO, the host can connect to {ProjectServer}, boot the installation media, and install the operating system.
```

Comment:
> Using this ISO, the host can connect to {ProjectServer}, boot the installation media, and install the operating system.

I would reverse the order:
1. Boot the installation media
2. Contact Foreman
3. Install the system